packages/strapi-utils/lib/convertRestQueryParams.js

```diff
@@ -0,0 +1,178 @@
+/**
+ * Converts the standard Strapi REST query params to a moe usable format for querying
+ * You can read more here: https://strapi.io/documentation/3.x.x/guides/filters.html
+ */
+
+const _ = require('lodash');
+
+const defaults = {
+  start: 0,
+  limit: 10,
+};
+
+/**
+ * Global convertor
+ * @param {Object} params
+ */
+const convertRestQueryParams = (params = {}) => {
+  if (typeof params !== 'object' || params === null) {
+    throw new Error(
+      `convertRestQueryParams expected an object got ${object === null ? 'null' : typeof object}`
+    );
+  }
+
+  let finalParams = _.assign({}, defaults);
+
+  if (Object.keys(params).length === 0) {
+    return finalParams;
+  }
+
+  if (_.has(params, '_sort')) {
+    _.assign(finalParams, convertSortQueryParams(params._sort));
```

Comment:
It could be better to use `Object.assign()`, I would love to see Lodash removed from the dependencies of Strapi.